grid_map_cv/CMakeLists.txt

```diff
@@ -29,6 +30,7 @@ set(dependencies
   sensor_msgs
   rclcpp
   OpenCV
+  pluginlib
```

Comment:
Hey thank you for the MR. Pluginlib should be an indirect dependency through the filters package, at least for ROS1. https://github.com/ros/filters/blob/lunar-devel/package.xml

Did you experience compilation issues?